issue-creator/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: CronJob
```

Comment:
That was what I originally intended to do (and may still be able to do), but Prow jobs can't use trusted secrets, like github access tokens with more than public_repo permissions, which the issue-creators might need. If I can avoid requiring a privileged access token then I'll be able to make this a Prow periodic job.